tools/@aws-cdk/cli-args-gen/lib/cli-type-gen.ts

```diff
@@ -129,9 +131,20 @@ function kebabToPascal(str: string): string {
     .join('');
 }
 
-function normalizeDefault(defaultValue: any): string {
-  if (typeof defaultValue === 'boolean' || typeof defaultValue === 'string') {
-    return String(defaultValue);
+function normalizeDefault(defaultValue: any, type: string): string {
+  switch (typeof defaultValue) {
+    case 'boolean':
+    case 'string':
+    case 'number':
+    case 'object':
+      return JSON.stringify(defaultValue);
```

Comment:
Evaluating nulls as 'object' still feels wrong to me but whatever I don't think it'd ever be a problem